src/doxygen/CGNSNav.dox

```diff
@@ -0,0 +1,44 @@
+/**
+ *
+ * \page CGNS_Navigation_Ill
+ * \section CGNS_Navigation_Ill CGNS File Navigation Illustration
+ * 
+ * To illustrate the use of the accessing a node routines, assume you have a file with CGNS
+ * index number \e filenum, a base node named \e Base with index number \e basenum, 2 zones
+ * (named \e Zone1 and \e Zone2, with indices 1 and 2), and user-defined data
+ * (\e User, index 1) below each zone. To move to the user-defined data node under zone 1,
+ * you may use any of the following:
+ *       
+ * \code{C}
+ *   cg_goto(filenum, basenum, "Zone_t", 1, "UserDefinedData_t", 1, NULL);
+ *   cg_goto(filenum, basenum "Zone1", 0, "UserDefinedData_t", 1, NULL);
+ *   cg_goto(filenum, basenum, "Zone_t", 1, "User", 0, NULL);
+ *   cg_goto(filenum, basenum, "Zone1", 0, "User", 0, NULL);
+ *   cg_gopath(filenum, "/Base/Zone1/User");
+ * \endcode
+ * Now, to change to the user-defined data node under zone 2, you may use the full path
+ * specification as above, or else a relative path, using one of the following:
```

Comment:
Maybe add a comment here that " The next relative cg_gorel/cg_gopath calls are valid only if cg_goto or cg_gopath with an absolute were used once before ! "